### PR TITLE
Add rating input at Create/Edit Product page

### PR DESCRIPTION
**Pull request by karolinakuzniewicz**
I want to merge this change because...

https://app.clickup.com/t/2549495/SALEOR-1384

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Translated strings are extracted.
1. [ ] Number of API calls is optimized.
1. [ ] The changes are tested.
1. [ ] Data-test are added for new elements.
1. [ ] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### Test environment config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1384 Add input to provide rating value in dashboard](https://app.clickup.com/t/2549495/SALEOR-1384)

File: schema.graphql
Comment (krzysztofwolski) [inline schema.graphql:7]: Do not apply formatting to schema.

**Review by krzysztofwolski — COMMENTED** [review]
Code is looking ok. Just revert the formatting of the schema.

File: schema.graphql
Comment (karolinakuzniewicz) [reply schema.graphql]: done

**Review by krzysztofwolski — APPROVED** [review]

File: src/products/utils/data.ts
Comment (orzechdev) [inline src/products/utils/data.ts:214]: We don't want to use `maybe` anymore, it was used when optional chaining wasn't available.

**Review by orzechdev — CHANGES_REQUESTED** [review]

File: src/products/utils/data.ts
Comment (karolinakuzniewicz) [reply src/products/utils/data.ts]: done

**Review by krzysztofwolski — APPROVED** [review]

**Review by orzechdev — APPROVED** [review]

**Comment by krzysztofwolski** [comment]
PR accepted. Please do not merge before the multichannel merge

**Review by jwm0 — APPROVED** [review]

**Pull request merged**
